example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -13,6 +13,7 @@
 
 #define CK_TILE_PIPELINE_COMPUTE 1
 #define CK_TILE_PIPELINE_MEMORY 2
+#define CK_TILE_PIPELINE_COMPUTE_V2 3
```

Comment:
Let's align those names with actuall pipeline name. 
```suggestion
#define CK_TILE_PIPELINE_COMPUTE_V4 3
```

Reply:
Solved.